### ISSUE TITLE
How to plot point data

### ISSUE THREAD

**Issue by lmullen** [sztanko/crosslet#8]
This looks like an amazing project. Thanks so much!

It wasn't clear to me from the documentation how to plot geocoded point data. Can you add that to the documentation?

**Comment by sztanko** [comment]
Hello there,

Unfortunately it is not really possible to plot points at this moment. I will keep this in mind on the next major release though. Thanks.

**Comment by diegovalle** [comment]
I have to agree this is an amazing project! For what it's worth I managed to create a [point map](http://diegovalle.github.io/tiger-mom-unam/crosslet.html) of test scores necessary to enter high schools in Mexico City by simply changing the line

```
_this.path = d3.geo.path().projection(_this.project);
```

to use pointRadius() to draw GeoJSON point geometries as circles

```
_this.path = d3.geo.path().projection(_this.project).pointRadius(function (d) {return Math.sqrt(d.properties.nstudents/7)+2});
```

d.properties.nstudents is the desired size of the circle
